Fhi.HelseId.Integration.Tests/TestFramework/Readme.md

```diff
@@ -0,0 +1,160 @@
+﻿# Testing guideline
+
+## General guideline
+
+- Treat tests as production code
+- It should be understandable what the tests do and follow AAA(Arange, Act, Assert) pattern
+- Test names should follow the [GWT(Given, When, Then)](https://martinfowler.com/bliki/GivenWhenThen.html), but without including GWT 
+
+```
+    [Test]
+    public async Task ConfiguredXXisTrue__CallTestApiEnpointPerson_ReturnYYinResponse()
+    {
+        // Arrange
+        var appsettingsConfig = new Dictionary<string, string?>
+        {
+            "XX", "true" },
+            
+        };
+        var testConfiguration = appsettingsConfig.BuildInMemoryConfiguration();
+        // Use testframeork application factory to override configurations and services
+        var appFactory = new TestWebApplicationFactory(testConfiguration, services =>
+        {
+            ..   
+        });
+
+        // Act
+        var client = appFactory.CreateClient();
+        var response = await client.GetAsync("api/person");
+
+        //Assert
```

Comment:
Nit: // Assert (will be caught by StyleCop when implemented)

Several occurrences in this file.

Reply:
Illustrating AAA pattern in readme guideline. Closing this.